src/dds_service.py

```diff
@@ -72,3 +74,315 @@ def builtin_observer(domain_id, dds_data, running):
                 dds_data.remove_endpoint(domain_id, sub)
 
     logging.info(f"builtin_observer({domain_id}) ... DONE")
+
+
+class dds_durability_kind(OrderedEnum):
```

Comment:
I wonder if it would be possible to define the ordering on the `Policy` classes directly. I really don't know Python well enough to know whether that's a good idea ... but it seems to be able to compare two, e.g., `Durability` policies or `Liveliness` policies.

(It it makes sense to do that, the ordering should probably not be defined for non-RxO QoS.)

Reply:
i think it should be possible to add the compare operator to the Policy classes.
Lets keep it here for now until all the other qos code from here found its way into the python wrapper?

Reply:
See above, I'm fine with going like this for now.